apitests/spec/device_telemetry_spec.ts

```diff
@@ -0,0 +1,131 @@
+import request = require('request');
+
+describe('devices api - ', () => {
+    describe('device telemetry - ', () => {
+        var req: request.RequestAPI<request.Request, request.CoreOptions, Object>;
+        // TODO: don't hardcode deviceId
+        var deviceId: string;
+        beforeAll(function (done) {
+            req = request.defaults({ json: true, baseUrl: 'https://localhost:44305/api/v1/telemetry' });
+            done();
+        });
+
+        it('1. Get alert history', (done) => {
+            req.get('alertHistory', (err, resp, result: AlertHistory) => {
+                expect(result).toBeTruthy();
+                expect(result.data).toBeTruthy();
+                expect(result.devices).toBeTruthy();
+                expect(result.maxLatitude).toBeTruthy();
+                expect(result.maxLongitude).toBeTruthy();
+                expect(result.minLatitude).toBeTruthy();
+                expect(result.minLongitude).toBeTruthy();
+                expect(result.totalAlertCount).toBeTruthy();
+                expect(result.totalFilteredCount).toBeTruthy();
+
+                expect(result.data[0]).toBeTruthy();
+                expect(result.data[0].deviceId).toBeTruthy();
+
+                expect(result.data[0].ruleOutput).toBeTruthy();
+                expect(result.data[0].timestamp).toBeTruthy();
+                expect(result.data[0].value).toBeTruthy();
+
+                expect(result.devices[0]).toBeTruthy();
+                expect(result.devices[0].deviceId).toBeTruthy();
+                expect(result.devices[0].latitude).toBeTruthy();
+                expect(result.devices[0].longitude).toBeTruthy();
+                expect(result.devices[0].status).not.toBeNull();
+                done();
+            });
+        }, 10000);
```

Comment:
Timeout is by default 5 secs for Jasmine tests, I had to increase it for some tests. Once we add retrying we could get rid of them.